data/10/measure_annotations.xml

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a reason not to have the facsimile infos in the MEI it belongs to?

Reply:
This is just a proposal. The zones have to be corrected manually anyway. If we want to use this data, I will merge it manually into the original MEI.